docker-compose.yml

```diff
@@ -147,9 +147,10 @@ services:
       FILE_UPLOAD_MAX_FILE_SIZE_BYTES: 15000000
       FILE_UPLOAD_USE_TEMP_FILES: "true"
       FILE_UPLOAD_TMP_PATH: "/tmp"
-      SESSION_MONGODB_URL: mongodb://mongodb:27017/fwa-sessions
+      SESSION_MONGODB_URL: mongodb://mongodb:27017/lpa-sessions
```

Comment:
perhaps not critical but we might as well have our own sessions collection - made it easier to understand some of the weirdness i was seeing when i could trace this in mongo locally